src/lib/ledger/selectTransport.js

```diff
@@ -0,0 +1,15 @@
+module.exports = isBrowser => {
+  if (isBrowser) {
+    const Transport = require('@ledgerhq/hw-transport-webusb').default
+    return Transport
+  } else {
+    if (process.env.NODE_ENV === 'test') {
+      // TODO: replace this with mock hardware interface
+      const Transport = require('@ledgerhq/hw-transport-node-hid').default
+      return Transport
+    } else {
```

Comment:
this doesn't look right to me
mocking should be done from tests code not the application code
by doing it this way, the code in the else branch will never get executed in tests, which means that the tests will not be able to detect bugs in it